TransactionService/docker_context/TransactionServiceApi.py

```diff
@@ -10,81 +9,92 @@
 
 
 @app.route(endpoints.quote_endpoint, methods=['GET'])
-async def getQuote(request, trans_num, user_id, stock_symbol):
-    data = {
-        'transaction_num': trans_num,
-        'user_id': user_id,
-        'stock_symbol': stock_symbol
-    }
+async def getQuote(request):
+    data = request.json
     res, err = validateRequest(data, quote_schema)
     if not res:
         return response.json(errorResult(err, data), status=400)
 
-    # quote logic would go here. try to keep api clean
+    data = app.config['serviceLogic'].getQuote(data['trans_num'], data['user_id'], data['stock_symbol'])
+
+    # Format RETURNS for all commands
 
     return response.json(data)
 
 
 # BUY ENDPOINTS ------------------------------------------------
 @app.route(endpoints.buy_endpoint, methods=['POST'])
 async def buyStock(request):
-    res, err = validateRequest(request.json, buy_schema)
+    data = request.json
+    res, err = validateRequest(data, buy_schema)
     if not res:
-        return response.json(errorResult(err, request.json), status=400)

```

Comment:
We passed in the request.json object here in case cause the validation checks to make sure the json object exists in the request

Reply:
reverted back to checking request.json for all except getQuote.